azurerm/internal/services/web/app_service_managed_certificate_resource.go

```diff
@@ -117,14 +117,19 @@ func resourceArmAppServiceManagedCertificateCreateUpdate(d *schema.ResourceData,
 	}
 	appServicePlanID = *appService.SiteProperties.ServerFarmID
 
+	appServicePlanIDParts, err := parse.AppServicePlanID(appServicePlanID)
```

Comment:
Really minor, but for the sake of convention, could you refactor the names of these vars for:
`appServicePlanID` => `appServicePlanIDRaw`
and
`appServicePlanIDParts` => `appServicePlanID`

Reply:
haha... I saw that when I was doing it last night... ill fix it now